terraform/variables.tf

```diff
@@ -97,14 +85,15 @@ variable "alerts" {
 
 variable "envvars" {
   type = map(string)
+  # FIXME: I removed these because I couldn't figure out how to override them from the deployment main.tf file without overwriting the whole object
   default = {
```

Comment:
Yeah, since its a map you can't override individual entries. You could convert this to an object but that might be overkill.

Reply:
Nah, I'll just leave these here as comments.